### ISSUE TITLE
Nordlond Bestiary doesn't update from 1.0.8 to 1.1.0 on The Forge

### ISSUE THREAD

**Issue by Farmeroz** [crnormand/gurps#1581]
Got the prompt to update Nordlond Bestiary module yesterday and it failed.  I tried several times, same result.  It attempts it (it takes while because of the size I assume) and then fails.  Tried again today - same result.

![image](https://user-images.githubusercontent.com/89975749/190137336-869e0eb2-df5c-4bfb-9ea1-1125fb4f455f.png)

I'm wondering if it's because of the size of it, perhaps it's exceeding my Data limit by trying to download it before purging the current version?  There's no error to indicate it, but I only have 152.4MB of free Data space at the moment.  How big is the update?  And if this is the issue, perhaps it can be logged with The Forge to point out it shouldn't do that?  And if it's not the problem...help! :-)

**Comment by Farmeroz** [comment]
1.1.0 is no longer showing on The Forge.

**Comment by mjeffw** [comment]
I'm sorry! I'll take a look tomorrow.

**Comment by Farmeroz** [comment]
No worries, and thanks :-)

**Comment by mjeffw** [comment]
Sorry it took so long!

The Nordlond Bestiary zip file, which is what is downloaded, is 14.1 MB. Unzipped and installed, it takes up 16.8 MB.

@Farmeroz What version of Foundry are you running on the Forge?

**Comment by mjeffw** [comment]
I took a deep breath, created a new world on the Forge, and upgraded to Foundry v10, GGA 0.15.0, and it updated Nordlond Bestiary to 1.1.0 with no problem.

@Farmeroz, if you are on Foundry v10 and GGA 0.15.0, then the update to Nordlond Bestiary should work. If it doesn't I think you have to contact Forge support to see what's going on.

I'll leave this open for a few more days to wait for your feedback.

**Comment by Farmeroz** [comment]
@mjeffw thanks for looking into it - I'm on V9 still (have been waiting for lots of modules to update/stabilise).  I don't current get an update option for Nordlond.  Was the issue perhaps that it was showing me 1.1.0 even though I was on V9 and that's now fixed (so I don't see it)?

I might do the V10 update and see how much it breaks everything :-)

**Comment by mjeffw** [comment]
@Farmeroz -- Nordlond Bestiary v1.1.0 is Foundry v10 only. There is no backwards compatibility. 👎  You should have never had the option to update to 1.1.0. Were you able to download version 1.0.8 of the bestiary?

**Comment by Farmeroz** [comment]
@mjeffw It did update to 1.0.8 and then it told me there was 1.1.0 and tried to update to that.  Then it stopped showing 1.1.0. when I checked back a couple of days later.  I've just updated to v10 and Nordlond has gone to 1.1.0 (and GGA to 0.15.3) so that seems all good.  Only downside is I now need to activate all my old modules and settings again - I wish Foundry/Forge would create update paths that didn't disable modules :-)

Thanks for your help on this!
